.changeset/weak-olives-confess.md

```diff
@@ -0,0 +1,6 @@
+---
+'@xyflow/react': patch
+'@xyflow/system': patch
+---
+
+fix(NodeResizer): handle undefined dimensions with default values
```

Comment:
are you sure we need this? the changeset tool is for managing contributions - maybe it's not required here

Reply:
You are right